test/cluster/object_store/test_backup.py

```diff
@@ -740,6 +740,12 @@ async def test_restore_with_streaming_scopes(manager: ManagerClient, object_stor
 
     schema, keys, replication_opts = await create_dataset(manager, ks, cf, topology, logger, num_keys=10000, min_tablet_count=512)
 
+    # make sure replicas propagated before doing the sanity check
+    await manager.api.tablet_repair(servers[0].ip_addr, ks, cf, 'all')
```

Comment:
Why do we need tablet_repair after create_dataset and before check_mutation_replicas?
Maybe create_dataset should use cl=ALL